www/src/utils/node/docs.js

```diff
@@ -165,48 +180,46 @@ exports.createPages = async ({ graphql, actions }) => {
     })
   })
 
+  packages.nodes.forEach(node => {
```

Comment:
So... does anyone know why this is necessary? It seems like this is handled in [node/packages.js](https://github.com/gatsbyjs/gatsby/blob/master/www/src/utils/node/packages.js#L45)? Are we accidentally creating double pages for each local package?